docker/updater/scheduler.sh

```
@@ -0,0 +1,47 @@
+mkdir -p /opt/updater/cache
+
+# Pull both the literal tag the operator's stack is pinned to AND
+# `:latest`. The first is useful for re-pulling exact-version updates
+# (in case of a tag move); the second is what surfaces "X.Y.Z available"
+# to the UI.
+#
+# We do not pull `:X.Y` (rolling minor) because operators who want patch
+# updates should still consent to them via the Apply button.
+pull_once() {
+  ts=$(date -u +%Y-%m-%dT%H:%M:%SZ)
+  for ref in "${NOWLEDGE_MEM_IMAGE}:latest"; do
```

Shellcheck (0.11.0)

[error] Since you double quoted this, it will not word split, and the loop will only run once.

(SC2066)